samples/nfc/writable_ndef_msg/src/ndef_file_m.c

```diff
@@ -30,29 +33,44 @@ static const uint8_t m_url[] = /**< Default NDEF message: URL "nordicsemi.com".
 	{'n', 'o', 'r', 'd', 'i', 'c', 's', 'e', 'm', 'i', '.', 'c', 'o', 'm'};
 
 /* Flash partition for NVS */
-#define NVS_FLASH_DEVICE FIXED_PARTITION_DEVICE(storage_partition)
+#define FLASH_DEVICE FIXED_PARTITION_DEVICE(storage_partition)
 /* Flash block size in bytes */
-#define NVS_SECTOR_SIZE  (DT_PROP(DT_CHOSEN(zephyr_flash), erase_block_size))
-#define NVS_SECTOR_COUNT 2
+#define SECTOR_SIZE  (DT_PROP(DT_CHOSEN(zephyr_flash), erase_block_size))
+#define SECTOR_COUNT 2
```

Comment:
Just to check, erase_block_size in reality doesn't make sense for ZMS to choose the sector_size. And its value is 4K I guess for most of the boards.
Did you make the calculations to verify that 4000 bytes (4096 - header)  are enough for your application ?
The recommended size should be at least double the size that you need for your data